### PR TITLE
Adding my thoughts on the juju getting started doc.

### PR DESCRIPTION
**Pull request by mbruzek**
Here are my thoughts. Please review.

### CODE REVIEW

File: docs/getting-started-guides/juju.md [hunk 1]
```diff
@@ -36,27 +36,32 @@ Juju dependencies for your platform.
...
 mkdir ~/.juju2
 sudo docker run -v ~/.juju2:/home/ubuntu/.local/share/juju -ti jujusolutions/charmbox:devel
 ```
 
-At this point from either path you will have access to the `juju
-bootstrap` command. However we will need to configure the credentials for the
- Juju cloud provider before we can proceed.
+### Configure Juju to point a cloud
+
+At this point you have access to the Juju client. If you wish to use a cloud,
```
Comment (lazypower) [inline docs/getting-started-guides/juju.md:54]: I really like this refactoring, but "if you wish to use a cloud" seems wrong in this context. They're going to _have_ to use a cloud unless they're on that very specific 1.25 local:kvm provider. Perhaps we should change just that sentence to "Before you can leverage juju client, you just first configure the credentials for the juju provider"

File: docs/getting-started-guides/juju.md [hunk 2]
```diff
@@ -91,24 +95,29 @@ etcd/0       active         idle        2.0-beta2 1                         54.1
...
+all available on the Kubernetes leader unit. Fetch the kubectl package so you
+can run commands on the new Kuberntetes cluster.
 
+Use the `juju status` command to figure out which Kubernetes unit is the leader
+and copy the file from the leader:  
 
 ```shell
 juju scp kubernetes/1:kubectl_package.tar.gz .
 tar xvfz kubectl_package.tar.gz
+kubectl --config config get pods
```
Comment (lazypower) [inline docs/getting-started-guides/juju.md:113]: there is no --config flag on kubectl

File: docs/getting-started-guides/juju.md [hunk 3]
```diff
@@ -197,28 +205,41 @@ We can add node units like so:
...
 ```shell
 ./kube-down.sh
 ```
 
-or destroy your current Juju environment (using the `juju env` command):
+If you want stop the servers you can destroy your current Juju environment
+(using the `juju env` command):
 
 ```shell
 juju destroy-environment --force `juju env`
```
Comment (lazypower) [inline docs/getting-started-guides/juju.md:228]: this is not proper for juju 2.0

juju kill-controller  `juju env`

File: docs/getting-started-guides/juju.md [hunk 4]
```diff
@@ -197,28 +205,41 @@ We can add node units like so:
...
+of Kubernetes.
 
- - [Bundle Repository](http://releases.k8s.io/{{page.githubbranch}}/cluster/juju/bundles)
-   * [Kubernetes master charm](https://releases.k8s.io/{{page.githubbranch}}/cluster/juju/charms/trusty/kubernetes-master)
-   * [Kubernetes node charm](https://releases.k8s.io/{{page.githubbranch}}/cluster/juju/charms/trusty/kubernetes)
+The Kubernetes layer and bundles can be found in the `kubernetes`
+project on github.com:
+
+ - [Bundle location](https://github.com/kubernetes/kubernetes/tree/master/cluster/juju/bundles)
+ - [Kubernetes charm layer location](https://github.com/kubernetes/kubernetes/tree/master/cluster/juju/layers/kubernetes)
```
Comment (lazypower) [inline docs/getting-started-guides/juju.md:242]: I'm not positive, but i think they're going to ding us on these URL's.

File: docs/getting-started-guides/juju.md [hunk 5]
```diff
@@ -236,5 +257,11 @@ works with [Amazon Web Service](https://jujucharms.com/docs/stable/config-aws),
...
-If you do not see your favorite cloud provider listed many clouds can be
-configured for [manual provisioning](https://jujucharms.com/docs/stable/config-manual).
+If you do not see your favorite cloud provider listed many clouds with ssh
+access can be configured for
+[manual provisioning](https://jujucharms.com/docs/stable/config-manual).
+
+The Juju Kubernetes work is curated by a very small group of community members.
+Let us know how we are doing. If you find any problems please open an
+[issue at the kubernetes project](https://github.com/kubernetes/kubernetes/issues)
+and tag it with "juju" so we can find it.
```
Comment (lazypower) [inline docs/getting-started-guides/juju.md:267]: perfect! :+1: :+1: :+1: :+1: :+1:   I give this single line a full five thumbs up and want to merge it based on this line alone!

**Comment by lazypower** [comment]
Some minor nits left as feedback, but I'm going to merge this and adjust as required. Thanks for the feedback and updates @mbruzek

**Comment by mbruzek** [comment]
Thanks for the review @chuckbutler 

I don't have any problem fixing those problems, but you merged the branch!

File: docs/getting-started-guides/juju.md (hunk 1 above)
Comment (mbruzek) [inline docs/getting-started-guides/juju.md:54]: I agree. Good word-smithing.

File: docs/getting-started-guides/juju.md (hunk 2 above)
Comment (mbruzek) [inline docs/getting-started-guides/juju.md:113]: Sorry it should be "--kubeconfig"

File: docs/getting-started-guides/juju.md (hunk 3 above)
Comment (mbruzek) [inline docs/getting-started-guides/juju.md:228]: Sorry, I was just trying to fix the "or..." You are correct to use 2.0

File: docs/getting-started-guides/juju.md (hunk 4 above)
Comment (mbruzek) [inline docs/getting-started-guides/juju.md:242]: These urls were to a different repository so the can not be relative or use the docs repo. I believe they dinged us on URLs when they still pointed to github/google/kubernetes/...

File: docs/getting-started-guides/juju.md (hunk 5 above)
Comment (mbruzek) [inline docs/getting-started-guides/juju.md:267]: moved it to the top

**Pull request merged**